packages/btc/README.md

```diff
@@ -309,6 +310,9 @@ interface SendUtxosProps {
   onlyConfirmedUtxos?: boolean;
   skipInputsValidation?: boolean;
   excludeUtxos?: BaseOutput[];
+
+  // EXPERIMENTAL: the below props are unstable and can be altered at any time
+  skipInputsValidation?: boolean;
```

Comment:
`skipInputsValidation` is duplicated.

Reply:
Fixed at: [a4ffe3c](https://github.com/ckb-cell/rgbpp-sdk/pull/150/commits/a4ffe3c4036cb673fbaa48c683098ea8b7434311)